lib/assert.js

```diff
@@ -299,7 +295,7 @@ function expectedException(actual, expected) {
     return false;
   }
 
-  if (Object.prototype.toString.call(expected) == '[object RegExp]') {
+  if (Object.prototype.toString.call(expected) === '[object RegExp]') {
```

Comment:
Importing it is probably better. If we ever move toward storing `Object.prototype.toString` in a variable to prevent tampering in userland code, it would require less updating.